worker-batch-ingestion-plugin/README.md

```diff
@@ -0,0 +1,10 @@
+# Ingestion Batch Worker Plugin
+
+## Configuration
+
+### Environment Variables
+
+- **CAF_STAGING_SERVICE_BASEPATH**
+  - The staging service base path, required.
```

Comment:
I see this says required, is this newly required or did it already exist? If it is new and is required I would think this constitutes a breaking change.

Reply:
It already existed, since I was adding doc I thought it should be mentioned.

Reply:
<img width="677" alt="image" src="https://user-images.githubusercontent.com/18459684/190167804-5fba5e4a-6d5c-4b4b-a4b3-48c9419bbffe.png">